core/views.py

```diff
@@ -14,6 +17,19 @@ def get_page_size(self, request):
             return self.max_page_size
         return super().get_page_size(request)
 
+# Permit to send feedback emails from users
+@api_view(['POST'])
+def feedback(request, format=None):
```

Comment:
```suggestion
@api_view(["POST"])
def feedback(request, format=None):
    """
    Send feedback emails from users
    """
```
Use Python [Docstrings](https://www.python.org/dev/peps/pep-0257/)